srcpkgs/pueue/template

```diff
@@ -1,32 +1,33 @@
 # Template file for 'pueue'
 pkgname=pueue
-version=2.1.0
+version=3.1.2
 revision=1
 build_style=cargo
 build_helper="qemu"
+make_install_args="--path pueue"
 short_desc="Command-line tool for managing long-running tasks"
 maintainer="crater2150 <void@qwertyuiop.de>"
 license="MIT"
 homepage="https://github.com/Nukesor/pueue"
 changelog="https://raw.githubusercontent.com/Nukesor/pueue/main/CHANGELOG.md"
-distfiles="https://github.com/Nukesor/pueue/archive/v${version}.tar.gz"
-checksum=cd5c6500e65960f6a102db5d0f0544f49eec8f74b2cc0df16ded8e2525a545f6
+distfiles="https://github.com/Nukesor/pueue/archive/refs/tags/v${version}.tar.gz"
+checksum=653eac9b7fc111cc4b9bddacbbf514932a8d273a059b20b1cc66af74e500eb5e
+make_check=no  # tests are nondeterministic
 
 case "$XBPS_TARGET_MACHINE" in
 	x86_64*|i686*|arm*|aarch64*) ;;
 	*) broken="ftbfs in ring" ;;
 esac
 
-post_build() {
+post_install() {
+	PUEUE="${DESTDIR}/usr/bin/pueue"
 	for shell in bash fish zsh; do
-		vtargetrun target/${RUST_TARGET}/release/pueue completions $shell .
+		vtargetrun ${PUEUE} completions $shell .
 	done
```

Comment:
there is no reason for the PUEUE variable, it's only used once

Reply:
Updated to use the value directly.